mos_tests/object_storage/conftest.py

```diff
@@ -0,0 +1,255 @@
+#    Copyright 2016 Mirantis, Inc.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+
+import logging
+import ntpath
+import os
+import random
+import tempfile
+import yaml
+
+import pytest
+from six.moves import configparser
+
+from mos_tests.functions import os_cli
+
+logger = logging.getLogger(__name__)
+
+
+@pytest.yield_fixture
+def admin_remote(fuel):
+    """SSH remote to nailgun node"""
+    with fuel.ssh_admin() as remote:
+        yield remote
+
+
+@pytest.yield_fixture(scope='module')
+def ctrl_remote(env):
+    """SSH remote to random controller"""
+    controller = random.choice(env.get_nodes_by_role('controller'))
+    with controller.ssh() as remote:
+        yield remote
+
+
+@pytest.fixture
+def openstack_client(ctrl_remote):
+    """Client to Openstack"""
+    return os_cli.OpenStack(ctrl_remote)
+
+
+@pytest.fixture
+def swift_client(ctrl_remote):
+    """Client to Swift"""
+    return os_cli.Swift(ctrl_remote)
+
+
+@pytest.fixture
+def s3cmd_client(ctrl_remote):
+    """Client to s3cmd tool"""
+    return os_cli.S3CMD(ctrl_remote)
+
+
+@pytest.yield_fixture
+def swift_create_container(swift_client):
+    """Creates container with Swift client"""
+    bucketname = 'TESTBUCKET{0}'.format(random.randint(0, 10000))
+    out = swift_client.post(bucketname)
+    yield bucketname, out
```

Comment:
That is right, but maybe someone else will need it later.
If all was correct during `swift_client.post(bucketname)` output of it will be empty.